### DISCUSSION TITLE
Help setting up PROXY_HEADER / REVERSE_PROXY_AUTH

### DISCUSSION THREAD

**Discussion by mtojay** [babybuddy/babybuddy#784] [Support]
I am in the process of using SSO for all my selfhosted services where its possible. In theory, and from my understanding, the implementation in babybuddy with the REVERSE_PROXY_AUTH is all i need. however i cant get it to work even on a really basic level. i am not lying, i spent at least a full workday tinkering with authentik and babybuddy by now but nothing. by now i am even at the point where i completely discarded the reversproxy. i try to pass the header in an api testing tool (in this case insomnia) by hand to babybuddy to make sure it works but still nothing. i have yet to see my babybuddy logs even acknowledge me trying to login with REVERSE_PROXY_AUTH - it just doesnt detect someone wanting to log in.

the header is set should be x_auth_user with a username. it just doesnt work. my current dockercompose:

version: "2.1"
services:
  babybuddy:
    image: lscr.io/linuxserver/babybuddy:latest
    container_name: babybuddy
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Berlin
      - CSRF_TRUSTED_ORIGINS=https://xxx.xxxxxx.xxx (obviously is not this -  just edited out)
      - REVERSE_PROXY_AUTH=true
    volumes:
      - ./data:/config
    ports:
      - 8098:8000
    restart: unless-stopped

in theory REVERSE_PROXY_AUTH should be activated right? i dont understand why it just doesnt accept my passed header or at least disallows it or tells me anything.
I feel like tearing my hairs out because i have so little info to troubleshoot with.  i also realize that not a lot of people use this feature, so that makes it a bit harder to troubleshoot aswell. 

if anyone who has made this work would be nice enough to give me a nudge in to the right direction i would be really really thankful.

**Comment by cdubz** [comment]
@mtojay sorry to hear this is so much trouble :frowning_face:

It does indeed seem like this is not a commonly used feature. Based on the docs I'm wondering -- are you also setting [`PROXY_HEADER`](http://docs.baby-buddy.net/configuration/security/#proxy_header)? I think this should be set as `PROXY_HEADER=HTTP_X_AUTH_USER` based on what you have described.

The documentation does seem a bit unclear on this... lets see if this work and then maybe we can improve the docs.
